test/models/ingest_job_test.rb

```diff
@@ -412,7 +406,10 @@ class IngestJobTest < ActiveSupport::TestCase
       studyAccession: @basic_study.accession,
       trigger: ann_data_file.upload_trigger,
       jobStatus: 'success',
-      numFilesExtracted: 1
+      numFilesExtracted: 1,
+      machineType: metadata_params.machine_type,
+      error: nil,
```

Comment:
See comment beginning "AIUI".

```suggestion
      action: 'ingest_cluster',
```